noetic/distribution.yaml

```diff
@@ -5424,6 +5424,23 @@ repositories:
       url: https://github.com/swri-robotics/mapviz.git
       version: master
     status: developed
+  marine_msgs:
+    doc:
+      type: git
+      url: https://github.com/apl-ocean-engineering/marine_msgs.git
+      version: 2.0.1
```

Comment:
The branch name here should typically be the same as the one in the `source` entry.  If you really don't want them to be the same they don't have to be, but that is recommended.

Reply:
Thanks, change commited.